server/models/database.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
```

Comment:
This is wild haha. I am not equipped to review this to the full extent, I would just make sure that you follow a tutorial for writing a script with flags haha

Reply:
Why do you say wild? I did look at a video and a bunch of pages.

Reply:
I say wild because the code is so hard to read but it is bash not you lol. In powershell you could do this:

```ps1
param(
    [switch]Create
    [switch]Delete
    ...
)

if($Delete) {
    // delete db command
}
# ... 
```

Notice that this reads a lot easier than the while true loop etc.

Reply:
Yup that looks way better.
In bash you either use `case` like I did or use `getopts` which is better for handling arguments.
Going forward, maybe I'll put PowerShell on my Ubuntu then just write PowerShell scripts instead while still using zsh. I switched from using PowerShell in console because some things are unnecessarily harder than with Unix shells (e.g., setting the executable bit on a file `chmod +x <filename>`. Maybe I just wasn't able to find the easiest way.

Reply:
Thank you!